canvas_modules/harness/src/client/components/sidepanel-properties.jsx

```diff
@@ -951,6 +969,8 @@ SidePanelModal.propTypes = {
 		light: PropTypes.bool,
 		useLightOption: PropTypes.func,
 		useEditorSize: PropTypes.func,
+		trimSpaces: PropTypes.bool,
+		setTrimSpaces: PropTypes.func,
```

Comment:
This isnt being passed in from App.js. im not able to toggle this side panel. You probably missed that change during rebase
<img width="248" alt="image" src="https://user-images.githubusercontent.com/1628100/215226031-b43d0a1d-9e73-43f1-83ee-5fe71ec3855d.png">